core/comp_ui.py

```diff
@@ -340,6 +354,10 @@ def __init__(
         # hash of matches -> count.  Has exactly ONE entry at a time.
         self.dupes = {}  # type: Dict[int, int]
 
+    def ReadlineInitCommands(self):
+        # type: () -> List[str]
+        return ['set horizontal-scroll-mode on']
```

Comment:
Please add a comment with this link, so we remember what it's for!

https://github.com/oils-for-unix/oils/issues/257

Reply:
Nice catch! Added a comment